scripts/utils/internals.js

```diff
@@ -100,6 +100,14 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     }
   }
 
+  function signHashWithPrivateKey(hash, privateKey) {
+    const Util = require("ethereumjs-util")
```

Comment:
`ethereumjs-util` is already imported as `ethUtil` in the same file.

Reply:
Great catch! ⚾ 🧤